hack/main.go

```diff
@@ -0,0 +1,14 @@
+package main
```

Comment:
Can we move these files under a different directory?  we mostly store scripts under `hack`

Reply:
Moved to `cmd/crd-gen`